__tests__/test-data.test.js

```diff
@@ -0,0 +1,62 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("General errors", () => {
+  test("404: responds with status code 404  with appropriate message when given a unavailable route", () => {
+    return request(app).get("/api/trea").expect(404);
```

Comment:
If you're testing this, you should test for the message you've sent back. At the moment, you're just testing express' 404.